aries/aries_vcx_core/src/wallet/askar/crypto_box.rs

```diff
@@ -0,0 +1,173 @@
+use sodiumoxide::crypto::{
```

Comment:
Just 1 general query, did you consider not using `sodiumoxide` for the sealedbox crypto? AFAIK Askar should have all of the sealedbox functionality we need. e.g.; https://github.com/hyperledger/aries-askar/blob/main/src/kms/envelope.rs

a little hesistant on `sodiumoxide` just because it doesn't seem super maintained, and requires the libsodium binary be packed/linked in. I believe askar's implementation is using a pure rust impl of crypto box (`crypto_box_rs`/`crypto_box`) https://crates.io/crates/crypto_box

anyway, something to consider. it could reduce complexity